tests/hi_test.py

```diff
@@ -40,3 +40,12 @@ def test_heating_degree_time_collection():
     assert isinstance(hourly_category, HourlyContinuousCollection)
     assert len(hourly_category.values) == calc_length
     assert hourly_category[4000] == 1
+
+
+def test_humidex():
+    """Test the humidex function."""
+    assert humidex(22, 15) == pytest.approx(25.96900, rel=1e-3)
+    assert humidex(20, 20) == pytest.approx(27.56977, rel=1e-3)
+    assert humidex(28, 20) == pytest.approx(35.56977, rel=1e-3)
+    assert humidex(28, 26) == pytest.approx(41.45899, rel=1e-3)
+    assert humidex(34, 25) == pytest.approx(46.33840, rel=1e-3)
```

Comment:
This test is good and, when you add a method to get the dicomfort category, it would also be good to have a test for that.

Reply:
Added with https://github.com/ladybug-tools/ladybug-comfort/pull/41/commits/18e393d435260f3eed9c53cd5c2963f5c3c46ddc